docs/source/tools/sandbox.rst

```diff
@@ -70,17 +70,22 @@ To start Sandbox with authentication based on `JWT <https://jwt.io/>`__ tokens,
 use one of the following command line options:
 
 - ``--auth-jwt-rs256-crt=<filename>``.
-  The sandbox will expect all tokens to be signed with RSA256 with the public key loaded from the given X.509 certificate file.
+  The sandbox will expect all tokens to be signed with RS256 (RSA DSA with SHA-256) with the public key loaded from the given X.509 certificate file.
```

Comment:
I'm sure you meant "DSA" to mean "digital signature algorithm", but it turns out it means ["Digital Signature Algorithm"](https://en.wikipedia.org/wiki/Digital_Signature_Algorithm), which is another signing algorithm unrelated to RSA.

[Auth0 refers to it as "RSA Signature with SHA-256".](https://auth0.com/docs/tokens/concepts/signing-algorithms) Perhaps we can just use their terminology.

```suggestion
  The sandbox will expect all tokens to be signed with RS256 (RSA signature with SHA-256) with the public key loaded from the given X.509 certificate file.
```